docs/getting-started/grouped-nav-ui.asciidoc

```diff
@@ -0,0 +1,19 @@
+[[grouped-nav-ui]]
+= Grouped navigation
+
+Grouped navigation is a new navigation UI for {elastic-sec} that groups related pages and highlights commonly visited areas for a streamlined experience. 
```

Comment:
Made this suggestion because "a new navigation UI" seems less concrete (like you're talking about a new navigation experience) than a new menu. However if that was your intention, feel free to ignore this edit.

Reply:
Made this suggestion because "a new navigation UI" seems less concrete (like you're talking about a new navigation experience) than a new menu. However if that was your intention, feel free to ignore this edit.